### PR TITLE
[VUMM-612] Make a backend to H2 Compatible and resolve Liquibase path issue after version upgrade

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

**Comment by anandjakhaniya** [comment]
@liam-verta  I have taken h2 dependencies from the UAC service. I don't know what I am doing wrong here. Could check it, please?

File: backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java [hunk 1]
```diff
@@ -94,6 +94,10 @@ public static String buildDatabaseServerConnectionString(RdbConfig rdb) {
       return rdb.DBConnectionURL;
     }
 
+    if (rdb.isH2()) {
+      return rdb.RdbUrl + ";DB_CLOSE_DELAY=-1";
```
Comment (jkwatson-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: any idea why we didn't need this in UAC?

Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: > any idea why we didn't need this in UAC?

I have tried to run uac without this and I am getting the error. To reproduce the issue. You have to start uac using the H2 database configuration.

Comment (jkwatson-verta) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: but... we run UAC without this on every PR, and we don't get an error?

Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: > but... we run UAC without this on every PR, and we don't get an error?

It surprised me too. If we are running uac service not test cases then I get the error..

Comment (jkwatson-verta) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: I just started it and it worked fine. 🤔

**Review by jkwatson-verta — APPROVED** [review]

File: backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java (hunk 1 above)
Comment (jkwatson-verta) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java:98]: I do see that this helps things in the registry though. 🤔

**Pull request merged**